### ISSUE TITLE
Using Ctrl+Tab, immediately switch tabs instead of showing a list

### ISSUE THREAD

**Issue by vlakoff** [WinMerge/winmerge#2324]
When doing Ctrl+Tab (or Ctrl+Shift+Tab) to switch tabs, all of a sudden I have now a "document list", and the tab is switched only when I release the keys. That behavior is absolutely unbearable to me. I was used to tabs just immediately switching when pressing the keys.

I don't know what happened, it was working as desired previously. I should have made a change by mistake. I have searched everywhere in the settings, in the context menus, etc. I don't find how to restore the good behavior.

Please, help me. Such "document switchers" are absolutely unbearable to me. I do need the "tab immediately switch" behavior. I really can't work because of this problem, and that document switcher is driving me insane right now.

**Comment by vlakoff** [comment]
Another thing, with my currently broken settings, the switch order is by "tab recency", instead of the good "left to right" order. Another behavior that I really can't stand…

I'm begging, please someone help me…

**Comment by sdottaka** [comment]
This feature was introduced in version 2.16.6, released on 2020-02-23, and it is always enabled without an option to disable it.
By quickly pressing and releasing the Ctrl+TAB keys, the list should not be displayed, and the behavior should be the same as in versions before 2.16.6.

**Comment by vlakoff** [comment]
Indeed, before changing computer, I was used to an old version of WinMerge.

I'm immediately reverting to WinMerge 2.16.5, and looking for replacement software.

**Comment by sdottaka** [comment]
If you use Ctrl+F6 instead of Ctrl+Tab, won't you get the behavior you expect?
